src/Mvc/Mvc.Api.Analyzers/src/AddResponseTypeAttributeCodeFixAction.cs

```diff
@@ -43,7 +44,14 @@ public AddResponseTypeAttributeCodeFixAction(Document document, Diagnostic diagn
 
         protected override async Task<Document> GetChangedDocumentAsync(CancellationToken cancellationToken)
         {
-            var context = await CreateCodeActionContext(cancellationToken).ConfigureAwait(false);
+            var nullableContext = await CreateCodeActionContext(cancellationToken).ConfigureAwait(false);
+            if (nullableContext == null)
+            {
+                return _document;
+            }
+
+            var context = nullableContext.Value;
```

Comment:
FYI this shouldn't be needed. The flow analysis should do the right thing after it sees the null check at L48.

Reply:
Example: https://github.com/aspnet/AspNetCore/pull/10375/files#diff-51adbcdc12291a2342784a001a9f502bR57

Reply:
```C#
error CS1061: 'AddResponseTypeAttributeCodeFixAction.CodeActionContext?' does not contain a definition for 'SymbolCache' and no accessible extension method 'SymbolCache' accepting a first argument of type 'AddResponseTypeAttributeCodeFixAction.CodeActionContext?' could be found
```

Doesn't seem to work if I remove it. Perhaps because it's a value type.